specifyweb/frontend/js_src/lib/localization/merging.ts

```diff
@@ -60,4 +60,14 @@ export const mergingText = createDictionary({
   dismissFromMerging: {
     'en-us': 'Dismiss from merging',
   },
+  agentContainsGroupDescription: {
+    'en-us':
+      'The following agents cannot be merged since they contain group members',
```

Comment:
add a colon `:` at the end of this string

---
OR:


(optional) it seems like you tried to make the solution generic, and not specific to agen table.
to that end, you can improve it a bit further:
instead of `The following agents cannot be merged since they contain group members`
use this string:
```
The following records cannot be merged. Reason:
Agents contain group members
```
and these would be two separate strings - the first one is not specific to Agents, thus can be reused. The second one will be just for agents.

of course, feel free to reword the string or ask Grant for better wording.

---

(optional) also, this string is a bit too long for a header in my opinion, so I would have displayed it as a `<p>` instead, but I would understand it if you chose `<H2>` just for styling

Reply:
Yeah I chose H2 because didn't want to think about styling it. We can switch to <p> and see how that works out.

@grantfitzsimmons 

> of course, feel free to reword the string or ask Grant for better wording.